client/my-sites/theme/main.jsx

```diff
@@ -166,7 +172,7 @@ const ThemeSheet = React.createClass( {
 						{ i18n.translate( 'Open Live Demo', { context: 'Individual theme live preview button' } ) }
 					</span>
 				</a>
-				{ this.getFullLengthScreenshot() && img }
+				{ screenshot && img }
```

Comment:
Just a bit confusing having this here. Maybe move it to declaration of `img` and use only `img` here. Not really important suggestion.